### PR TITLE
Create index and columns

### LINKED ISSUE #4518: [Term Entry] Python:Pandas dataframe index
### Reason/inspiration (optional)

"We would like a new term entry in the `dataframe` concept for [Pandas](https://www.codecademy.com/resources/docs/pandas): index. The entry should go in a new file under `docs/content/pandas/concepts/dataframe/terms/index/index.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use the trace function to find the trace of a square matrix.
- Should contain information for functions and syntax.
- Should have a Python codebyte with a working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### LINKED ISSUE #4519: [Term Entry] Python:Pandas dataframe columns
### Reason/inspiration (optional)

"We would like a new term entry in the `dataframe` concept for [Pandas](https://www.codecademy.com/resources/docs/pandas): columns. The entry should go in a new file under `docs/content/pandas/concepts/dataframe/terms/columns/columns.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use the trace function to find the trace of a square matrix.
- Should contain information for functions and syntax.
- Should have a Python codebyte with a working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by 0-Brandon**
### Description
Added entries for the .index and .columns attributes of DataFrames. 
### Issue Solved

Closes #4518
Closes #4519

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Tags:
+  - 'Pandas'
+  - 'Attributes'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.index`** attribute represents the row labels (index) of the DataFrame. It is an Index object and can be used to view or assign new values to the row labels.
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:16]:
```suggestion
The **`.index`** attribute represents the row labels (index) of the DataFrame. It returns an Index object and can be used to view or assign new values to the row labels.
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## Syntax
+
+```pseudo
+Dataframe.index
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:21]:
```suggestion
pandas.DataFrame.index
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+## Example
+
+In the following example, the `.index` attribute is used to modify the row labels of the `monthlyCost` DataFrame:
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:26]:
```suggestion
In the following example, the `.index` attribute is used to view and modify the row labels of the `monthlyCost` DataFrame:
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+data = {'April': [88, 92, 79, 85],
+'May': [94, 77, 90, 78],
+'June': [89, 80, 95, 81]}
+
+monthlyCost = pd.DataFrame(data)
+
+# Printing the DataFrame
+
+print(monthlyCost)
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:42]:
```suggestion
import pandas as pd

# Creating the DataFrame representing monthly cost of 4 items

data = {'April': [88, 92, 79, 85],
'May': [94, 77, 90, 78],
'June': [89, 80, 95, 81]}

monthlyCost = pd.DataFrame(data)

# Print the DataFrame
print(f'{monthlyCost} \n')

# modify the row labels
monthlyCost.index = ['bread', 'milk', 'apples', 'onions']

# print the modified monthlyCost Dataframe
print(f'{monthlyCost} \n')

# print the row label or index of monthlyCost DataFrame
print(monthlyCost.index)
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+The output for the above code is as follows:
+
+```shell
+    April    May    June
+0    88       94       89
+1    92       77       80
+2    79       90       95
+3    85       78       81
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:52]:
```suggestion
  April  May  June
0     88   94    89
1     92   77    80
2     79   90    95
3     85   78    81 

        April  May  June
bread      88   94    89
milk       92   77    80
apples     79   90    95
onions     85   78    81 

Index(['bread', 'milk', 'apples', 'onions'], dtype='object')
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+The above code produces the following output:
+
+```shell
+        April    May      June
+bread    88       94       89
+milk     92       77       80
+apples   79       90       95
+onions   85       78       81
+```
+
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:84]:
```suggestion
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+Codebyte Example
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:85]:
```suggestion
## Codebyte Example

```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+The following Codebyte demonstrates how to view and update a DataFrame using the `.index` attribute:
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:86]:
```suggestion
The following Codebyte Example demonstrates how to view and update a DataFrame using the `.index` attribute:
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+```codebyte/python
+import pandas as pd
+
+# Sample DataFrame
+data = {'c1': [1, 2], 'c2': [3, 4]}
+df = pd.DataFrame(data)
+
+print(df)
+
+df.index = ['r1', 'r2']
```
Comment (letsbuildanything) [inline content/pandas/concepts/dataframe/terms/index/index.md:97]:
```suggestion
# updating the row label
df.index = ['r1', 'r2']
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@0-Brandon 
Thank you for contributing to Codecademy Docs!
I've requested below some changes. Please make them.

**Review by letsbuildanything — APPROVED** [review]
Ready for second round of review!

**Comment by Sriparno08** [comment]
Hey @0-Brandon, please open a new PR for the `.index` entry as we only want one logical change in a PR. In case of this PR, make a new commit removing the `.index` entry and leaving the `.columns` entry as it is.

File: content/pandas/concepts/dataframe/terms/columns/columns.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+Tags:
+  - 'Pandas'
+  - 'Attributes'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.columns`** attribute represents the column labels of the DataFrame. It returns an Index object and can be used to view or assign new values to the column labels.
```
Comment (avdhoottt) [inline content/pandas/concepts/dataframe/terms/columns/columns.md:16]:
```suggestion
The **`.columns`** attribute represents the column labels of the DataFrame. It returns an `Index` object and can be used to view or assign new values to the column labels.
```

File: content/pandas/concepts/dataframe/terms/index/index.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+Tags:
+  - 'Pandas'
+  - 'Attributes'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.index`** attribute represents the row labels (index) of the DataFrame. It returns an Index object and can be used to view or assign new values to the row labels.
```
Comment (avdhoottt) [inline content/pandas/concepts/dataframe/terms/index/index.md:16]:
```suggestion
The **`.index`** attribute represents the row labels (index) of the DataFrame. It returns an `Index` object and can be used to view or assign new values to the row labels.
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @0-Brandon, I've suggested a minor change. Please make it and let me know when you are done.

**Review by avdhoottt — APPROVED** [review]
LGTM!!

**Pull request merged**